packages/mjml-social/src/Social.js

```diff
@@ -3,7 +3,7 @@ import { BodyComponent } from 'mjml-core'
 export default class MjSocial extends BodyComponent {
   static allowedAttributes = {
     align: 'enum(left,right,center)',
-    'border-radius': 'unit(px)',
+    'border-radius': 'string',
```

Comment:
unit(px,%) would be better here ?

Reply:
🤦‍♂ Yep, you're 100% right - updated. Thanks!